scripts/release/debian/deb_install.sh

```diff
@@ -81,13 +84,22 @@ setup() {
             exit 1
         fi
     fi
-    echo "deb [arch=$(dpkg --print-architecture)] https://packages.microsoft.com/repos/azure-cli/ ${CLI_REPO} main" \
-        > /etc/apt/sources.list.d/azure-cli.list
+
+    if [ -f /etc/apt/sources.list.d/azure-cli.sources ]; then
+        rm /etc/apt/sources.list.d/azure-cli.sources
```

Comment:
The content of `/etc/apt/sources.list.d/azure-cli.sources` is overwritten by `tee /etc/apt/sources.list.d/azure-cli.sources` anyway.

We should remove `/etc/apt/sources.list.d/azure-cli.list`, not `/etc/apt/sources.list.d/azure-cli.sources`.

Reply:
indeed. fixed.